### ISSUE TITLE
BREAKING CHANGE: neovim-nightly -> neovim... opinions wanted

### ISSUE THREAD

**Issue by mjlbach** [nix-community/neovim-nightly-overlay#96]
When I first made this overlay, I wanted to allow users to install 0.4.4 and nightly side by side. I went half of the way there, but instead stopped short and only renamed the package neovim-nightly, but not the binary. This means it's currently not possible to have both nvim-nightly and nvim installed simultaneously, defeating the purpose.

In an effort to a align with core, I'd like to rename neovim-nightly to neovim in the flake. What this will mean, is all users will need to change neovim-nightly to neovim in their personal configurations. I realize this is a breaking change, so we can provide both definitions for an intermediate time period.

The goal with this is to use the same flake provided with core.

@teto thoughts?

**Comment by teto** [comment]
I dont get the advantage of having different names, you could only get the first `nvim` in PATH anyway ?
When rebuilding, it's a matter of swapping either neovim attribute name or overlay.

**Comment by mjlbach** [comment]
Right, currently we do have different names, which functionally is useless because they are both on path as nvim. The alternative to the status quo would be to rename the binary to nvim-nightly, so both would be on path. As it stands, there is no benefit to the current usage of a different name

**Comment by teto** [comment]
yeah that's what I thought. So either you move the binary in a postInstall hook while keeping a different attribute name or you change the name. I tend to use vim/vscode as a backup when I blow up my neovim config so you may want someone else opinion.

**Comment by mjlbach** [comment]
(you already know this but stating for others)

The principal disadvantage of keeping the attribute name as nvim-nightly and adding a postInstall hook, is I'll have to continually modify the flake from core on each change, and people will have to alias nvim to nvim-nightly to keep their current installs working. The main advantage is people can install stable nvim from nixpkgs in parallel with nvim-nightly.
